packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -12,30 +12,45 @@ import {DateTimePicker} from '../atoms/dateTimePicker'
 import {InfoColor} from '../atoms/infoMessage'
 
 export interface PublishArticlePanelProps {
-  initialPublishDate?: Date
+  publishedAtDate?: Date
+  updatedAtDate?: Date
+  publishAtDate?: Date
   pendingPublishDate?: Date
+  publishBehaviorDate?: boolean
```

Comment:
Might be clearer if you can rename this. `hasPublishDate` or `isPublishDateActive `